### PR TITLE
Task-a3dzK6St

### PR DESCRIPTION
**Pull request by AntonShalenui**
https://trello.com/c/a3dzK6St/11-change-header-text

### CODE REVIEW

**Comment by AntonShalenui** [comment]
https://trello.com/c/a3dzK6St/11-change-header-text

**Comment by BYavorskyj** [comment]
Where header.css file?

**Comment by AntonShalenui** [comment]
sorry, give me a minute

File: src/client/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import './app.css';
+
```
Comment (BYavorskyj) [inline src/client/App.js]: extra space

**Pull request merged**